test/args-test.ts

```diff
@@ -76,6 +77,106 @@ test('Args smoke test', (assert) => {
   parent.firstName = "Thomas";
 });
 
+test('Rendering args derivatives', (assert) => {
```

Comment:
This is a nitpick but I would prefer test names that are more descriptive, so they are more accessible to future contributors. For example, perhaps instead of "Rendering args derivatives", we could say something like "Tracked properties that depend on `args` re-render correctly". The terse names make it difficult to understand at a glance the difference between the two tests.